clj-refactor.el

```diff
@@ -36,6 +36,7 @@
 (require 'yasnippet)
 (require 'paredit)
 (require 'multiple-cursors-core)
+(require 'cl-macs)
```

Comment:
Is this still needed?

Reply:
Quite unlikely. It has been `cl-lib` for a very long time.

Reply:
clj-refactor is using cl-case, cl-incf, and cl-assert all of which are not present in cl-lib but are present in the cl-macs wrapper.